### ISSUE TITLE
Variable<Element>  not anymore subset of ObservableType Why

### ISSUE THREAD

**Issue by mendesbarreto** [ReactiveX/RxSwift#563]
Hello Everyone!

I downloaded a example made in RxSwift last week and I started a new project based on it. 

The project I had downloaded was made in RxSwift 2.2.0 but I decided to get the new version the RxSwift and I saw the Variable class not anymore a subset of Observable Type, 

because this example bellow not working:

```
func <-> <T>(property: ControlProperty<T>, variable: Variable<T>) -> Disposable {
   let bindToUIDisposable = variable
      .bindTo(property)
   let bindToVariable = property
      .subscribe(onNext: { n in
         variable.value = n
         }, onCompleted:  {
            bindToUIDisposable.dispose()
      })
   
   return StableCompositeDisposable.create(bindToUIDisposable, bindToVariable)
}
```

This example has some workaround?

**Comment by kzaher** [comment]
Hi @mendesbarreto ,

We've removed that before we've published 2.0.0 version so you probably didn't use 2.2.0 version but some beta version before :)

You just need to say `asObsevable`.

```
func <-> <T>(property: ControlProperty<T>, variable: Variable<T>) -> Disposable {
    let bindToUIDisposable = variable.asObservable()
        .bindTo(property)
    let bindToVariable = property
        .subscribe(onNext: { n in
            variable.value = n
        }, onCompleted:  {
            bindToUIDisposable.dispose()
        })

    return StableCompositeDisposable.create(bindToUIDisposable, bindToVariable)
}
```

**Comment by mendesbarreto** [comment]
Thx man! 

Everything is working heheheh
